app/views/hub/forced_password_resets/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="slab">
+  <div class="grid__item width-one-half shift-one-fourth">
+    <div class="form-card form-card--compact">
+      <div class="form-card__header">
+        <h1 class="form-card__title">Please update your password. </h1>
+        <p class="text--help">It’s important that you keep this password secure and do not share it with anyone.
+          Please keep all client information safe and confidential.</p>
+      </div>
+      <%= form_for(@user, builder: VitaMinFormBuilder, url: { action: :update }, html: { method: :put }) do |f| %>
+        <div class="form-card__content">
+          <%= f.cfa_input_field(:password, "New password", type: "password", classes: ['form-width--long', 'password']) %>
```

Comment:
`I8n.t('general.views.passwords.edit.new_password')` would do the job here (if you feel like marking strings for translation).

Tip: You can search the codebase for some English copy and see if exists in `en.yml` then right-click in RubyMine and do `Copy reference` and get an I18n key.